ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly-4.16-upgrade-from-eus-4.14.yaml

```diff
@@ -110,6 +110,25 @@ tests:
           memory: 200Mi
     - chain: openshift-upgrade-qe-test-eus
     workflow: cucushift-installer-rehearse-gcp-ipi-cco-manual-workload-identity
+- as: gcp-ipi-workers-rhel8-fips-f28
```

Comment:
sounds like it could cover both gcp and rhel, shall we drop existed gcp profile to keep one gcp only?
```
- as: gcp-ipi-oidc-rt-fips-f14
```

Reply:
I am okay with it, @shellyyang1989 wdyt?

Reply:
I'm okay with it too.

Reply:
Done